<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDeviceControl</class>
 <widget class="QWidget" name="frmDeviceControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labNodeID">
     <property name="text">
      <string>编  号: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labPositionID">
     <property name="text">
      <string>位  号: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labDeviceName">
     <property name="text">
      <string>控制器: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labNodeName">
     <property name="text">
      <string>变量: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labNodeType">
     <property name="text">
      <string>类  型: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labNodeSign">
     <property name="text">
      <string>气  种: </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
